includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -433,8 +435,40 @@ public function getData() {
 		return $data;
 	}
 
+	private function getSitenameData() {
+		$sitename = $this->wg->Sitename;
+
+		if ( $wgSitenameForComscoreForWikiId = WikiFactory::getVarValueByName( 'wgSitenameForComscore', $this->wikiId ) ) {
```

Comment:
It's not the nicest way to do the comparison by writing an assignment in condition. 😕